string process_id
---
int32 nb_holders
